src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const Books = () => {
+  const booksArray = [{
+    id: 1, title: 'Dune', author: 'Frank Herbert', genre: 'Science Fiction',
+  }];
+  return (
+    <>
+      <Navbar />
+      <h1 className="books-page-title">E-Bookopolis</h1>
+      <form id="form">
+        <input type="text" id="book-title" name="book-title" className="book" placeholder="Book Name..." />
+        <select name="categories" id="categories-dropdown">
+          <option value="" disabled selected hidden>Categories</option>
```

Comment:
- [ ] I would recommend you pay attention to the logged warning and implement the suggestions as they help to write more conventional code.

  ![hammas-selected](https://user-images.githubusercontent.com/36542986/154993748-3d9e2a14-4a3e-438c-9440-b845e56f338a.jpg)

Reply:
I have resolved this error.